FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -65,18 +65,18 @@ void Maslow_::begin(void (*sys_rt)()) {
   axisTRHomed = false;
   axisTLHomed = false;
 
-  tlX = 25.4253399079432;
```

Comment:
I'm really stoked that this is gone and that it is in settings now. That is great work